pkgs/top-level/aliases.nix

```diff
@@ -1116,6 +1116,7 @@ mapAliases {
   opendylan_bin = throw "opendylan_bin has been removed from nixpkgs as it was broken"; # Added 2024-07-15
   openelec-dvb-firmware = throw "'openelec-dvb-firmware' has been renamed to/replaced by 'libreelec-dvb-firmware'"; # Converted to throw 2024-10-17
   openethereum = throw "openethereum development has ceased by upstream. Use alternate clients such as go-ethereum, erigon, or nethermind"; # Added 2024-05-13
+  openexr_3 = openexr; # Added 2025-03-12
```

Comment:
Is there some policy or convention that decides if a versioned package could stay inside `pkgs`?

I would imagine that, in the future, there might be `openexr_4`, and that `openexr_3` would reach the end of its lifecycle. Shouldn't we leave `openexr_3` inside `pkgs` and let `openexr = openexr_3`?

Reply:
> and let openexr = openexr_3?

you mean openexr_4 here? If yes, I'd say yes, that's what we should do, only because that is what is generally done in nixpkgs as far as I know. We only keep openexr_2 because we don't have any choice, but we would get rid of it as soon as no other package depends on it, what do you think?

It's always difficult to only have one version of widely used library in a package set. In the geospatial set of packages, we have the same "issue" with geos for instance.